utils/math-utils.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Get the random choice for the players
+ * @param CHOICES 
+ * @returns ROCK (or) PAPER (or) SCISSOR
+ */
+export const getRandomChoice = (CHOICES) => CHOICES[Math.floor(Math.random() * CHOICES.length)];
```

Comment:
could you let me know if this random could be reused anywhere ? 
example in some other scenario, in the same app, if we have another random number needed how could that be achieved ?

Reply:
Yes, this will work for any array that we pass as a parameter to the getRandomChoice function